tests/src/Functional/MessageEntityDelete.php

```diff
@@ -223,7 +229,7 @@ public function testReferencedEntitiesDelete() {
     $message->save();
 
     $account->delete();
-    $this->assertFalse(Message::load($message->id()), 'Message deleted after deleting single referenced user.');
+    $this->assertEquals(4, $delete_queue->numberOfItems(), 'Message queued for deletion after deleting single referenced user.');
   }
```

Comment:
I went a slightly different route, reverting back to the previous assertion logic, but adding in the cron runs that will process the queue items. Thinking about it a bit more, ultimately what we want to see is the deletion of the items, so I'm not sure how valuable it is to see items going into the queue as much as seeing the messages being deleted.